Classes/helpers/class.tx_caretaker_LatestVersionsHelper.php

```diff
@@ -75,10 +77,40 @@ public static function updateLatestTypo3VersionRegistry()
             if (is_array($details) && !empty($details['stable'])) {
                 $stable[$major] = $details['stable'];
             }
+            if (is_array($details) && is_array($details['releases'])) {
+			    $found = false;
+			    $latestCheckedVersion = '';
+			    foreach ($details['releases'] as $version => $versionDetails) {
+			        $latestCheckedVersion = $version;
+                    // if major version > latest_lts check if bugfix part of version number == '0';
+                    // in that case this is the security update and stop searching
+                    if (version_compare($major, $latestLts, '>')) {
+                        if ($versionDetails['type'] === 'security') {
+                            $security[$major] = $version;
+                            $found = true;
+                            break;
+                        }
+                        // if versionDetails->type === 'security' then this is the security update; stop searching
+                        if (self::extractBugfixNumberFromVersion($version) === 0) {
```

Comment:
Any reason why you don't use \TYPO3\CMS\Core\Utility\VersionNumberUtility::convertVersionStringToArray?